libs/txnbuild/setup_credit.go

```diff
@@ -38,6 +40,7 @@ func (setupCredit *SetupCredit) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for SetupCredit initialises the vtxnbuild struct from the corresponding xdr Operation.
```

Comment:
FromXDR for SetupCredit initialises the vtxnbuild struct from the corresponding **XDR** Operation.